### ISSUE TITLE
Example does not work

### ISSUE THREAD

**Issue by Thijxx** [miurahr/py7zr#405]
I try to run the example from the README but it returns an error:

```
    with py7zr.SevenZipFile('/tmp/myfile.7z', mode='r') as z:
        z.extractall(path=my_path)
```

The error returned is: `TypeError: __init__() missing 1 required positional argument: 'message'`.

**Comment by miurahr** [comment]
Please follow bug report template. please give information following.

Environment (please complete the following information):

OS: [e.g. Windows 10, Ubuntu Linux 18.04.01]
Python [e.g. 3.6, pypy3.6.9-7.3.0]
py7zr version: [e.g. v0.5b1, commit #123456 on master]

**Comment by miurahr** [comment]
This looks like a known bug in v0.17.0 that fixed in v0.17.1

https://github.com/miurahr/py7zr/blob/master/Changelog.rst

**Comment by Thijxx** [comment]
OS: Ubuntu 21.10
Python: 3.9
py7zr: 0.17.2

Unfortunately I cannot reproduce the error anymore. I have used `pyunpack` to handle my archive.
